netket/sampler/autoreg.py

```diff
@@ -57,11 +57,19 @@ def __repr__(self):
 class ARDirectSampler(Sampler):
     """
     Direct sampler for autoregressive neural networks.
-
-    `ARDirectSampler.machine_pow` has no effect. Please set the model's `machine_pow` instead.
     """
 
     def __pre_init__(self, *args, **kwargs):
+        """
+        Construct an autoregressive direct sampler.
+
+        Args:
+            hilbert: The Hilbert space to sample.
+            dtype: The dtype of the states sampled (default = np.float64).
+
+        Note:
```

Comment:
does Note get rendered correctly by sphinx?

Reply:
It turns out that in our Sphinx theme 'Note' must be the last section in a docstring, otherwise it will render other sections like 'Args' into 'Note'